cluster/gce/gci/configure-helper.sh

```diff
@@ -1039,6 +1063,15 @@ function start-kube-proxy {
   if [[ -n "${CLUSTER_IP_RANGE:-}" ]]; then
     sed -i -e "s@{{cluster_cidr}}@--cluster-cidr=${CLUSTER_IP_RANGE}@g" ${src_file}
   fi
+}
+
+# Starts kube-proxy pod.
```

Comment:
Can you update the comment here something like:

Starts kube-proxy static pod?

Otherwise, it is confusing to the reader why we need to prepare-kube-proxy-manifest-variables in two methods.

Reply:
Sure, done.